### ISSUE TITLE
Update form error example to show correct accessibility settings

### ISSUE THREAD

**Issue by Blackbaud-TrevorBurch** [blackbaud/skyux-errors#32]
https://developer.blackbaud.com/skyux/components/form#error-states

We should be showing the use of `aria-describedby` when a form error is shown. We also should update the error message to use the new status indicator component in order to fix accessibility issues with the old css psuedo selector error icon.

However, we need to show smart usage of `id`s and so we should hold off on this until we finish and release an `id` creation service. That work is tracked at blackbaud/skyux-core#65

[AB#1510576](https://dev.azure.com/blackbaud/Products/_workitems/edit/1510576)
